org.eclipse.xtext.xtext.ui/src/org/eclipse/xtext/xtext/ui/wizard/project/AdvancedNewProjectPage.xtend

```diff
@@ -211,22 +233,11 @@ class AdvancedNewProjectPage extends WizardPage {
 			}
 		}
 
-		val dependend = #[createUiProject, createIdeaProject, createWebProject]
-		if (!createIdeProject.selection && dependend.exists[selection]) {
-			val affectedProjects = dependend.filter[selection].join(', ', [text])
-			if (createIdeProject === source) {
-				reportIssue(ERROR, '''
-				Frontend projects like '�affectedProjects�' depends on '�createIdeProject.text�' project.
-				Please <a>deselect</a> these.''', [
-					dependend.forEach[selection = false]
-				])
-			} else {
-				reportIssue(ERROR, '''
-				Projects like '�affectedProjects�' depends on '�createIdeProject.text�' project.
-				Please <a>enable '�createIdeProject.text�'</a> project.''', [
-					createIdeProject.selection = true
-				])
-			}
+		if (autoSelectIdeProject) {
+			autoSelectIdeProject = false
+			reportIssue(INFORMATION, '''
+				'�createIdeProject.text�' project was automatically selected as option '�(source as Button).text�' requires it.
```

Comment:
This overrides the message about the buildsystem, but the statusWidget can only show one message at once. For me that does not sound like a big issue but maybe other see that different.